src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -340,8 +357,18 @@
                                 <tr>
                                     <td class="font-normal">{{ profiler_dump(authenticator.stub) }}</td>
                                     <td class="no-wrap">{{ source('@WebProfiler/Icon/' ~ (authenticator.supports ? 'yes' : 'no') ~ '.svg') }}</td>
+                                    <td class="no-wrap">{{ authenticator.authenticated is not null ? source('@WebProfiler/Icon/' ~ (authenticator.authenticated ? 'yes' : 'no') ~ '.svg') : '' }}</td>
```

Comment:
on other profiler pages, those svg are colored, can we have this also here?

Reply:
The color is set when the svg is inside a `.metrics` tag. This logic may be expanded to other classes in all profilers.

Since it's not related to this PR, I updated my PR to color these svg in the Security panel only.